source/addonHandler/__init__.py

```diff
@@ -751,17 +751,23 @@ def getCodeAddon(obj=None, frameDist=1):
 	# Not found!
 	raise AddonError("Code does not belong to an addon")
 
+
+translationFunctions = ["_", "ngettext", "pgettext", "npgettext"]

```

Comment:
This should be a constant (i.e. all upper case using underscores as word separators), it makes sense to make it private to this module with leading underscore, there is also no point for it to be mutable so tuple instead of a list. For an alternative, IMHO nicer, implementation see below.

Reply:
Agree on that, will fix it.